_pages/technical-details.markdown

```diff
@@ -62,9 +64,45 @@ of a Semantic Web-based information system as smooth as possible.
 There is also a [library](https://github.com/ledsoft/jopa-spring-transaction) for integrating JOPA with the declarative
 transaction management of Spring (think `@Transactional`) that is used in TermIt.
 
-#### JB4JSON-LD
+##### JB4JSON-LD
 [JB4JSON-LD](https://github.com/kbss-cvut/jb4jsonld) (Java Binding for JSON-LD) is a library for seamless marshalling and unmarshalling of POJOs into and from JSON-LD.
 It integrates with Jackson, so the set-up for an application is a matter of just a couple of lines of code.
 
 
-# Data Model
+## Data Model
+
+Vocabularies modelled in TermIt are using two specific data models -- Data description ontology and TermIt ontology, both developed by Knowledge-based and Software Systems Group (KBSS).
+
+### Data description ontology
+
+Basic ontology used for description of any data is called Data description ontology and is described on [this page](http://onto.fel.cvut.cz/ontologies/slovnik/agendovy/popis-dat/current/index-en.html). It is based on the Unified Foundational Ontology (UFO).
+
+The current version of Data description ontology can be found [here](https://onto.fel.cvut.cz/ontologies/page/slovnik/agendovy/popis-dat/model/verze/1.0.1) in turtle or RDF/XML.
+
+Following sections describes basic classes defined in the model.
+
+#### Vocabulary
```

Comment:
Maybe a diagram would be more informative than this extract of classes. Example would also help IMO.

Reply:
Full documentation of a model is included via link. I was thinking about diagram, but full diagram is too complicated and simple does not connect chosen classes directly.